tastyworks/models/order.py

```diff
@@ -97,8 +114,11 @@ def from_dict(cls, input_dict: dict):
         Parses an Order object from a dict.
         """
         details = OrderDetails(input_dict['underlying-symbol'])
+        details.order_id = input_dict['id'] if 'id' in input_dict else None
```

Comment:
Neater to just replace this with `details.order_id = input_dict.get('id')`. The `.get()` method returns a `None` by default if the key wasn't found